examples/spark/spark-worker-controller.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 metadata:
   name: spark-worker-controller
 spec:
-  replicas: 3
+  replicas: 1
```

Comment:
I mean, 3 is certainly kind of arbitrary, but it at least shows more than one worker running when you spin up the example.